### ISSUE TITLE
Error Action CreateAccount

### ISSUE THREAD

**Issue by Okhotnik-V** [syntifi/near-java-api#7]
Nice lib) 
Creating an account does not work.

[https://explorer.testnet.near.org/transactions/DSRgJocq3kLm5KXVC6m4zMG4D5857N1Auy9Q1DwPNp1N](https://explorer.testnet.near.org/transactions/DSRgJocq3kLm5KXVC6m4zMG4D5857N1Auy9Q1DwPNp1N)

**Comment by Okhotnik-V** [comment]
If it works for you, please provide a code on how to do it)

**Comment by Ctzzhang** [comment]
which action is createAccount? I can not find it

**Comment by Okhotnik-V** [comment]
> which action is createAccount? I can not find it

[https://docs.near.org/docs/concepts/transaction#action](https://docs.near.org/docs/concepts/transaction#action)

[https://github.com/syntifi/near-java-api/blob/main/src/main/java/com/syntifi/near/api/model/transaction/Action.java](https://github.com/syntifi/near-java-api/blob/main/src/main/java/com/syntifi/near/api/model/transaction/Action.java)

` //Transactions
        List<Action> list = new ArrayList<>();
        list.add(CreateAccountAction.builder().build());

        list.add(AddKeyAction.builder()
                .publicKey(publicKey2)
                .accessKey(accessKey)
                .build());

        list.add(TransferAction.builder().deposit(deposit).build());

        TransactionAwait encodedHash = TransactionService.sendTransactionAwait(
                nearService,
                signer1,
                receiver,
                publicKey,
                privateKey,
                list
        );`

**Comment by Ctzzhang** [comment]
> > which action is createAccount? I can not find it
> 
> https://docs.near.org/docs/concepts/transaction#action
> 
> https://nomicon.io/RuntimeSpec/Actions#addkeyaction
> 
> https://github.com/syntifi/near-java-api/blob/main/src/main/java/com/syntifi/near/api/model/transaction/Action.java
> 
> `
> 
> ```
>     List<Action> list = new ArrayList<>();
>     list.add(CreateAccountAction.builder().build());
>     list.add(AddKeyAction.builder()
>             .publicKey(publicKey2)
>             .accessKey(accessKey2)
>             .build());
>     list.add(TransferAction.builder().deposit(deposit).build());
> 
>     TransactionAwait encodedHash = TransactionService.sendTransactionAwait(
>             nearService,
>             signer1,
>             receiver,
>             publicKey,
>             privateKey,
>             list
>     );
> ```
> 
> `

how to create a top level account  ? I cannot find the function CreateAccountOnlyByRegistrar()

**Comment by Okhotnik-V** [comment]
Just write in String receiver = "example.testnet" and that's it.  But the problem is that you need to make the access keys manually.  At the moment I don't know how to do it.  I hope the developer of the library will create such a function)

**Comment by GarveyChang** [comment]
> which action is createAccount? I can not find it



> which action is createAccount? I can not find it

I don't know how to generate ed25519 keypair, take public key and hex encode it,can you provide a way to generate addresses via by 12 mnemonics?

**Comment by GarveyChang** [comment]
> Nice lib) Creating an account does not work.
> 
> https://explorer.testnet.near.org/transactions/DSRgJocq3kLm5KXVC6m4zMG4D5857N1Auy9Q1DwPNp1N

I don't know how to generate ed25519 keypair, take public key and hex encode it,can you provide a way to generate addresses via by 12 mnemonics?

**Comment by AB3rtz** [comment]
yes, it is here [AccountService](https://github.com/syntifi/near-java-api/blob/main/near-java-api-rpc/src/main/java/com/syntifi/near/api/rpc/service/AccountService.java), with example here  [AccountServiceTest](https://github.com/syntifi/near-java-api/blob/main/near-java-api-rpc/src/test/java/com/syntifi/near/api/rpc/service/AccountServiceTest.java).
